client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/list-item.js

```diff
@@ -1,125 +0,0 @@
-/** @format */
-
-/**
- * External dependencies
- */
-import React from 'react';
-import { expect } from 'chai';
-import { configure, shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-
-/**
- * Internal dependencies
- */
-import Button from 'components/button';
-import { CarrierAccountListItem } from '../list-item';
-import CarrierIcon from '../../../components/carrier-icon';
-import Dialog from 'components/dialog';
-
-configure( { adapter: new Adapter() } );
```

Comment:
No need to do this, the `tests/client/setup-test-framework.js` already takes care of this.